cql3/statements/create_index_statement.cc

```diff
@@ -32,6 +32,7 @@
 
 #include <boost/range/adaptor/transformed.hpp>
 #include <boost/algorithm/string/join.hpp>
+#include <stdexcept>
```

Comment:
Again no patch changelog. If you want a review, the patches should explain what they are trying to accomplish, and the reviewer can then verify that this goal is worthwhile and separately that the change indeed accomplishes that goal. The reviewer does not have to reverse engineer the intent.

Reply:
I think the changelog is fine now. I wouldn't have used the word "unintuitive" in the description - it's actually intuitive, just different from what our internal representation looks like. But I understood what you meant.